dev-java/openjdk/openjdk-17.0.5_p8.ebuild

```
@@ -46,11 +46,11 @@ SRC_URI="
 		$(bootstrap_uri x86 ${X86_XPAK})
 		$(bootstrap_uri riscv ${RISCV_XPAK})
 	)
-	riscv? ( https://dev.gentoo.org/~arthurzam/distfiles/dev-java/openjdk/openjdk-17.0.3-riscv.patch.xz )
+	riscv? ( https://github.com/felixonmars/archriscv-packages/raw/master/java17-openjdk/java17-riscv64.patch )
```

Please use a stable URL for the patch

```suggestion
	riscv? ( https://raw.githubusercontent.com/felixonmars/archriscv-packages/24658ef25980d72c7493d99c35922ab24f4a7799/java17-openjdk/java17-riscv64.patch )
```